fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/scripts/update_optuna.py

```diff
@@ -0,0 +1,146 @@
+import os
```

Comment:
let's not include this file as well, we are working anyway on organizing those utilities into a separate repo

Reply:
Removed